spec/index.html

```diff
@@ -1492,17 +1542,14 @@ <h2>Security Considerations</h2>
 
 <section id="internationalization" class="appendix informative">
   <h2>Internationalization Considerations</h2>
-  <p>RDF is restricted to representing string values with left-to-right or right-to-left direction indicators.
-    RDF provides a mechanism for specifying the language associated with
-    a string (<a>language-tagged string</a>),
-    but does not provide a means of indicating the base direction of the string.</p>
-
   <p>Unicode provides a mechanism for signaling direction within a string
-    (see [[[UAX9]]] [[UAX9]]),
-    however, when a string has an overall base direction which cannot be determined by the
+    (see [[[UAX9]]] [[UAX9]]).
+    RDF provides a mechanism for specifying the <a>base direction</a>
+    of a <a>directional language-tagged string</a>
+    to signal the initial text direction of a string.
+    When a string has an overall base direction which cannot be determined by the
     beginning of the string, an external indicator is required,
-    such as the [[HTML]] <a data-cite="HTML/dom.html#the-dir-attribute">dir attribute</a>,
-    which currently has no counterpart for <a>RDF literals</a>.</p>
+    such as the [[HTML]] <a data-cite="HTML/dom.html#the-dir-attribute">dir attribute</a>.</p>
```

Comment:
Not just that: strings which can be detected still need bidi isolation. In many formats (and particularly HTML) this is done by applying the base direction to the inserted string value. That is, the `dir` attribute does more than just setting the paragraph direction and saving the user-agent the trouble of finding the first-strong character.

The implication of this text is that only some strings need base direction. In practice many natural language strings would benefit from the metadata (not just the RTL or corner-case ones)

Reply:
Only literals having a language tag could benefit from having a base direction, for which we create a new datatype: `rdf:dirLangString`. By definition, base direction is not necessary for a literal with any other datatype.

In the case of RDF, there is substantial data in the wild that does not include any notion of base direction. Based on what you say, arguably, new data should consider the use of directional language-tagged strings rather than simple language-tagged strings.

Is there some suggested change for this language?

Reply:
The challenge here is that this implies that only strings with indeterminate direction need base direction metadata. I agree that only language strings need base direction and I18N's suggestion would be that, in general, it's better to use the new `dirLangString` to just `langString`. I recognize that `langString` has to continue to exist.

A different, perhaps more correct, formulation might be something like:

```suggestion
For most human language strings, but particularly for those
whose base direction cannot be accurately determined from the 
string content, is it valuable to have an external indicator in order
to get the proper display and isolation of the value.
One example of such an indicator is
the [[HTML]] <a data-cite="HTML/dom.html#the-dir-attribute">dir attribute</a>.
```

You might also consider adding a pointer to String-Meta here?